client/state/activity-log/actions.js

```diff
@@ -29,6 +31,8 @@ import {
 	REWIND_BACKUP_DISMISS_PROGRESS,
 } from 'state/action-types';
 
+requireHandlers( handlers );
```

Comment:
You know, this could be called from the action creator, which would be an even lazier load. It wouldn't introduce the handlers until just before the action is dispatched.

Reply:
the worry with that is the asynchronous loading nature. if we only lazy-load it at the action creator then we are _guaranteed_ to dispatch the action before the handler is loaded (event loop processing)

on the other hand it does give us reason to split out action creators when there is a module with lots of them but only a couple which require a specific handler set